packages/driver/src/cypress/stack_utils.ts

```diff
@@ -12,6 +12,8 @@ import { getStackLines, replacedStack, stackWithoutMessage, splitStack, unsplitS
 
 const whitespaceRegex = /^(\s*)*/
 const customProtocolRegex = /^[^:\/]+:\/{1,3}/
+// string containing `webpack:///./`
+const webpackDevtoolNamespaceRegex = /webpack:\/{3}\.\//
```

Comment:
I had to tweak this a bit in the original spike due to custom-namespaces. For example, this is Next's: 
![Screen Shot 2022-09-22 at 12 52 27 PM](https://user-images.githubusercontent.com/25158820/191817608-ed69a466-d3aa-45e6-b4c2-3aa58326367b.png)

You can see the namespace show up in the relative path e.g. `N_E_/pages/index.cy.js`. The original regex was `/webpack:\/\/(.*)?\.\//` to account for this, did this not work out?

Maybe we can use [output.devtoolModuleFilenameTemplate](https://webpack.js.org/configuration/output/#outputdevtoolmodulefilenametemplate) to enforce consistency across frameworks.

Reply:
@ZachJW34 Ah, that makes more sense now. The issue I ran into was that the regex you crafted broke relative paths if a spec was opened from outside the project root like `../e2e/specName.cy.ts`. I see what the regex intent was now, I'll take a look to see if there's a way to craft one that scrubs the custom namespace without killing relative pathing

Reply:
@ZachJW34 I've adjusted the regex and it seems to work as expected for Next.js apps (as well as for the system test that failed with the original pattern)